e2e/build.gradle

```diff
@@ -11,8 +11,8 @@ java {
 
 dependencies {
     //client
-    implementation 'org.apache.httpcomponents.client5:httpclient5:5.2.2'
-    implementation 'org.apache.httpcomponents.client5:httpclient5-fluent:5.2.2'
+    implementation 'org.apache.httpcomponents.client5:httpclient5:5.2.1'
```

Comment:
If we changed our code, do we need to also downgrade?  Or can we keep the latest version?

Reply:
For the import, we can keep the old code. But once we upgrade again, we will have to make the same changes as the header import won't be supported. Overview of the [package](https://hc.apache.org/httpcomponents-core-5.2.x/current/httpcore5/apidocs/overview-summary.html)